src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/regelmodell/resultat/BeregningsgrunnlagPeriode.java

```diff
@@ -165,6 +165,10 @@ public void setGrenseverdi(BigDecimal grenseverdi) {
         this.grenseverdi = grenseverdi;
     }
 
+    public boolean getHarVærtBesteberegnet() {
```

Comment:
Vi burde være mer konsistent ift get og set på boolske verdier getHarVærtBesteberegnet vs isBesteberegnet vs erBesteberegnet.
Litt lengre ned isSplitteATFLToggleErPå
Syns at vi burde bruke erBesteberegnet som vi ble enig om i fpsak for en del siden.